app/src/main/java/com/codesoom/assignment/exception/CatToyNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package com.codesoom.assignment.exception;
+
+public class CatToyNotFoundException extends RuntimeException{
+    public static final String DEFAULT_MESSAGE = "식별자가 [%d]인 고양이 장난감은 찾을 수 없습니다.";
```

Comment:
시점을 두고 잘 생각하셔야 해요. "그것은 찾을 수 없습니다"는 언제나 찾을 수 없다는 의미로 읽힐 수 있습니다.
```suggestion
public class CatToyNotFoundException extends RuntimeException{
    public static final String DEFAULT_MESSAGE = "식별자가 [%d]인 고양이 장난감을 찾을 수 없었습니다.";
```